index.Rmd

```diff
@@ -12,9 +12,6 @@ documentclass: book
 
 This book is intended to quickly introduce users to TERRA REF data through a series of tutorials. TERRA REF has many types of data, and most can be  accessed in multiple ways. Although this makes it more complicated to learn (and teach!), the objective is to provide users with the flexibility to access data in the most useful way.
 
-
-## Contents
```

Comment:
using the section collapsing in the table of contents will collapse all of the sections except for the ones in the index.Rmd (I'm not sure why honestly). I condensed the number of top level headers in this chapter to make the TOC styling look better